tools/src/publish-packages/tasks/grantTeamAccessToPackages.ts

```diff
@@ -22,32 +22,41 @@ export const grantTeamAccessToPackages = new Task<TaskArgs>(
     // so let's get all team members and check if they all are declared as maintainers.
     // If they don't, grant access for the team.
     const teamMembers = await Npm.getTeamMembersAsync(Npm.EXPO_DEVELOPERS_TEAM_NAME);
-    const packagesToGrantAccess = parcels.filter(
-      ({ pkgView, state }) =>
-        (pkgView || state.published) && doesSomeoneHaveNoAccessToPackage(teamMembers, pkgView)
-    );
+    const packagesToGrantAccess = parcels
+      .filter(filterPackagesToGrantAccess(teamMembers))
+      .map(({ pkg }) => pkg.packageName);
 
     if (packagesToGrantAccess.length === 0) {
       logger.success('\n🎖  Granting team access not required.');
       return;
     }
 
-    if (!options.dry) {
-      logger.info('\n🎖  Granting team access...');
+    logger.info(
+      `\n🎖  ${options.dry ? 'Team access would be granted to' : 'Granting team access to'}`,
+      packagesToGrantAccess.map((name) => green(name)).join(' ')
+    );
 
-      for (const { pkg } of packagesToGrantAccess) {
-        logger.log('  ', green(pkg.packageName));
-        await Npm.grantReadWriteAccessAsync(pkg.packageName, Npm.EXPO_DEVELOPERS_TEAM_NAME);
+    if (!options.dry) {
```

Comment:
I think that when `dry` flag is set we should have concise log info for every step is not performed, but would be otherwise 🤔

Reply:
That's how it works here, take a look a few lines above 😄